cgmanifest.json

```diff
@@ -490,6 +490,17 @@
         }
       }
     },
+    {
+      "component": {
+      "type": "other",
+      "other": {
+        "name": "cloud-init-vmware-guestinfo",
```

Comment:
Super minor, and will get cleaned up the next time we run a full cgmanifest regeneration, but these should ideally be in sorted order.

Reply:
Disregard, I misread this